package.json

```diff
@@ -30,12 +30,11 @@
   },
   "devDependencies": {
     "babel-plugin-external-helpers": "^6.22.0",
-    "babel-preset-env": "^1.6.0",
+    "babel-preset-env": "^1.7.0",
     "babel-register": "^6.26.0",
-    "prettier": "^1.7.4",
+    "prettier": "^1.14.3",
     "rollup": "^0.50.0",
-    "rollup-plugin-babel": "^3.0.2",
-    "rollup-watch": "^4.3.1"
```

Comment:
but why?

Reply:
rollup-watch is deprecated and uses the insecure dependency described in #169